### PR TITLE
minikube default installation using Docker can be used for TiDB

### PR DESCRIPTION
**Pull request by yahonda**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Fix https://github.com/pingcap/docs-tidb-operator/issues/1509
- Removed Note section
- Update startup message using minikube v1.24.0

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-tidb-operator/issues/1509

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/get-started.md [hunk 1]
```diff
@@ -125,28 +121,22 @@ minikube start
...
-🔑  The 'hyperkit' driver requires elevated permissions. The following commands will be executed:
-
-    $ sudo chown root:wheel /Users/user/.minikube/bin/docker-machine-driver-hyperkit
-    $ sudo chmod u+s /Users/user/.minikube/bin/docker-machine-driver-hyperkit
-
-
-💿  Downloading VM boot image ...
-    > minikube-v1.10.0.iso.sha256: 65 B / 65 B [-------------] 100.00% ? p/s 0s
-    > minikube-v1.10.0.iso: 174.99 MiB / 174.99 MiB [] 100.00% 6.63 MiB p/s 27s
+😄  minikube v1.24.0 on Darwin 12.1
```
Comment (DanielZhangQD) [inline en/get-started.md:124]: Should we make it clear in L106 for the minimum version to use?

Comment (yahonda) [reply en/get-started.md:124]: Good question.  I can suggest users to use the newer versions of minikube like v1.24.0 but have no idea which minikube version is minimum requirement. I mean users can run TiDB on minikube using default configuration.

Comment (DanielZhangQD) [reply en/get-started.md:124]: OK, let's just suggest users to use the newer versions, thanks!

Comment (yahonda) [reply en/get-started.md:124]: Added another commit to suggest users to use the newer versions.

File: en/get-started.md [hunk 2]
```diff
@@ -101,13 +101,9 @@ You are now ready to deploy TiDB Operator.
...
 You can create a Kubernetes cluster in a VM using [minikube](https://minikube.sigs.k8s.io/docs/start/), which supports macOS, Linux, and Windows.
 
-> **Note:**
->
-> minikube supports `--vm-driver=none` that uses a host Docker instead of a VM. However, this function is not yet fully tested on TiDB Operator and may result in operational failures. If you want to try TiDB Operator in a non-virtualized system (for example, in a VPS), consider using [kind](#create-a-kubernetes-cluster-using-kind) instead.
-
 Before deployment, make sure the following requirements are satisfied:
 
-- [minikube](https://minikube.sigs.k8s.io/docs/start/): version 1.0.0 or later versions. minikube requires a compatible hypervisor. For details, refer to minikube installation instructions.
+- [minikube](https://minikube.sigs.k8s.io/docs/start/): version 1.0.0 or later versions. We recommend upgrading to the newer versions like v1.24. minikube requires a compatible hypervisor. For details, refer to minikube installation instructions.
```
Comment (ran-huang) [inline en/get-started.md:106]:
```suggestion
- [minikube](https://minikube.sigs.k8s.io/docs/start/): version 1.0.0 or later versions. Newer versions like v1.24 is recommended. minikube requires a compatible hypervisor. For details, refer to minikube installation instructions.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM. Thank you!

**Comment by yahonda** [comment]
Updated as suggested.

**Review by ran-huang — APPROVED** [review]

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**